docs/theme.rst

```diff
@@ -28,7 +28,21 @@ If you want to create your own theme, see :doc:`theme/custom`.
    :local:
 
 Switching Themes
-----------------
+================
+
+In a nutshell
+-------------
+
+To use a theme, just source its file in your shell configuration file,
+after having load the liquidprompt, and then call `lp_theme`:
```

Comment:
This should be double backtics:
And, to make the following a code block (and therefore ignore spelling issues), you need two colons:
```suggestion
after having load the liquidprompt, and then call ``lp_theme``::
```